### DISCUSSION TITLE
GitHub student migration

### DISCUSSION THREAD

**Discussion by kiwamijapan** [community/community#36532] [GitHub Education]
Some time ago I set up this account, and attached my university e-mail address to it, so I can use student package. Recently, I set up second, more professionally looking account, and wanted to migrate my student package over there, so I've disconnected my e-mail from this account, yet when I try to connect it to the new one, I get message saying that student package was already used on other (this) account, whereas I've deleted it, even got notification that this e-mail won't be any longer connected with this account.
Is there a chance that I could still use student package on new account or am I stuck with old on this one?

**Comment by ossanna16** [comment]
Hey there @kiwamijapan and thank you for asking a good question! I'm glad you found our community 🙂
